GoogleForCreators/web-stories-wp
# Editor: Layer panel redesign

## Context

The layer panel was redesigned: [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235).

## Summary

Updates the layer panel to match the new designs.

## Relevant Technical Choices

- `LayerIcon` and `LayerContent` are only used in the layer panel, so editing those shouldn't break anything in the editor.
- I tried to keep everything consistent so I am sharing a `<LayerText />` component for the text content of each row.

## To-do

nothing

## User-facing changes

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/109732242-10063180-7b7a-11eb-8539-b3169db54411.png)|![new-layer-panel](https://user-images.githubusercontent.com/22185279/109849641-d7646780-7c0e-11eb-8783-c6a0ac6c53a1.gif)|



## Testing Instructions

1. Go to editor
2. Add one of each element type to the canvas and check out the layers in the layer panel

### QA






This PR can be tested by following these steps:

1. Go to editor
2. Add one of each element type to the canvas and check out the layers in the layer panel

### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6210
